docs/concepts/core-workflow-concepts.md

```diff
@@ -6,47 +6,45 @@ This document explains the core concepts that are needed to understand the workf
 
 ## Workflow
 
-A workflow is the definition of a process. It is made up of states and transitions between those states. The following diagram shows a simple workflow representing the lifetime of an item in a catalogue:
+A workflow is the definition of a process. It is made up of states and transitions between those states. The following diagram shows a  workflow representing a simple expense claim process:
 
 ```
-                  Create
-                     |
-                     | 
-                     |
-             +-------v-------+
-             |               |
-             | Waiting for   |
-      +------+ documentation |
-      |      |               |
-      |      +-------+-------+
-      |              |
-Edit                 | Publish
-(incomplete)         |
-      |      +-------v-------+               +---------------+
-      |      |               |               |               |
-      +------+   Published   |  Deprecate    |  Deprecated   |
-             |               +--------------->    (end)      |
-      +------>               |               |               |
-      |      +--+---------+--+               +---------------+
-      |         |         |
- Edit +---------+         | Delete
- (complete)               |
-             +------------v--+
-             |               |
-             |    Deleted    |
-             |     (end)     |
-             |               |
-             +---------------+
+                              +--------------+  Cancel      +--------------+
+                              |              +-------------->              |
+                              | In progress  |              |   Deleted    |
+                       +------>   (start)    <-------+      |    (end)     |
+                       |      |              |       |      |              |
+                  Edit |      +--+----+------+       |      +--------------+
+                       |         |    |              |
+                       +---------+    | Submit       |
+                                      |              | Reject
+                              +-------v------+       |
+                              |              |       |
+        Approve (claim >£500) | Waiting for  |       |
+        +---------------------+ approval     +-------+
+        |                     |              |
+        |                     +-------+------+
+        |                             |
+        |                             | Approve (claim <£500)
+        |                             |
++-------v------+              +-------v------+              +--------------+
+| Waiting for  |              |              |              |              |
+| senior       |   Approve    | Waiting for  |   Paid       |    Paid      |
+| approval     +--------------> payment      +-------------->    (end)     |
```

Comment:
No Reject path out of the `Waiting for senior approval` state?

Reply:
Nope. It's all about the title and being seen to be doing something. There's no real power there.